Some update

tugsbayasgalan opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #124664
* __->__ #124450
* #124449



cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang

Differential Revision: [D56440611](https://our.internmc.facebook.com/intern/diff/D56440611)

tugsbayasgalan committed:
Some update

[ghstack-poisoned]

tugsbayasgalan committed:
Update on "Some update"




[ghstack-poisoned]

gmagogsfm commented on .ci/docker/ci_commit_pins/executorch.txt:
```diff
@@ -1 +1 @@
-d4b3e5cc607e97afdba79dc90f8ef968142f347c
+554cd2792cf684818b9f33106eb1a6d19bb1f85bq
```
?

tugsbayasgalan replied:
not needed

gmagogsfm commented on torch/jit/_trace.py:
```diff
@@ -799,138 +970,26 @@ def forward(self, x):
-    return traced
+        traced_mod = WrapperMod(traced_func)
 
+    return torch.export.export(traced_mod, export_example_inputs, example_kwarg_inputs, strict=False).module()
```
how about some correctness verification logic between traced vs exported program?

tugsbayasgalan commented:
@tugsbayasgalan has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D56440611).

tugsbayasgalan commented:
@tugsbayasgalan has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D56440611).

tugsbayasgalan commented:
@tugsbayasgalan has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D56440611).

tugsbayasgalan commented:
@tugsbayasgalan has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D56440611).

tugsbayasgalan committed:
Update on "Some update"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang

Differential Revision: [D56440611](https://our.internmc.facebook.com/intern/diff/D56440611)

[ghstack-poisoned]

tugsbayasgalan committed:
Update on "Some update"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang

Differential Revision: [D56440611](https://our.internmc.facebook.com/intern/diff/D56440611)

[ghstack-poisoned]

tugsbayasgalan committed:
Update on "Some update"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang

Differential Revision: [D56440611](https://our.internmc.facebook.com/intern/diff/D56440611)

[ghstack-poisoned]
